security/security-file-token-provider.1.rst

```diff
@@ -121,12 +122,22 @@ the following is inserted (if not overridden) to the token parameters for the ge
 
 ::
 
-  "display_name": service-name
+  "display_name": token-service-name
   "no_parent":    true
   "policies":     [ "edgex-service-service-name" ]
 
-Note that ``display_name`` may be used by ``go-mod-secrets``
-as a hint for locating service secrets.
+Note that ``display_name`` is set by vault to be "token-" + the specified display name.
+This is hard-coded into ``token_store.go`` in Vault and cannot be changed.
```

Comment:
You don't need to mention where this is hard-coded I guarantee you exactly 0.1 microseconds after you merge this it will become out-of-date :smile: Just delete this line, but it's okay to still mention it's hard-coded here.

Reply:
Made statement less-specific.

Reply:
@bnevis-i it doesn't look like this line was changed at all...

for clarity, here is what I would propose:

```suggestion
This is currently hard-coded into in Vault version X.Y.Z and cannot be changed.
```

Reply:
Checked my modification in this time.